packages/ui/src/app/pages/WorkingGroups/WorkingGroup/WorkingGroup.stories.tsx

```diff
@@ -49,7 +89,17 @@ export default {
       return {
         accounts: parameters.isLoggedIn ? { active: { member: alice } } : { list: [{ member: alice }] },
 
-        // chain: undefined,
+        chain: {
+          tx: {
+            membershipWorkingGroup: {
+              addOpening: {
+                event: 'OpeningCreated',
+                onSend: args.onCreateOpening,
+                failure: parameters.createOpeningFailure,
+              },
+            },
+          },
+        },
 
         queryNode: [
```

Comment:
i think `membership` would be `args.isLead ? bob : alice,` this is because the `leader` has been changed to `{ membershipId: bob.id, isActive: true },` meaning bob is now the WG Lead